examples/time_frequency/plot_time_frequency_erds.py

```diff
@@ -111,3 +113,85 @@
     fig.colorbar(axes[0].images[-1], cax=axes[-1])
     fig.suptitle("ERDS ({})".format(event))
     fig.show()
+
+##############################################################################
+# Similar as for :class:`~mne.Epochs` objects, we can also export data from
+# :class:`~mne.time_frequency.EpochsTFR` and
+# `~mne.time_frequency.AverageTFR` objects to a
+# :class:`Pandas DataFrame <pandas.DataFrame>`:
+
+df = tfr.to_data_frame()
+df.time = df.time / 1000  # time to s
```

Comment:
```suggestion
df = tfr.to_data_frame(time_format=None)
```
This should make the time column already in seconds.